src/Cake.DocFx/DocFxBuildRunner.cs

```diff
@@ -51,6 +52,16 @@ private ProcessArgumentBuilder GetArguments(FilePath configFile, DocFxBuildSetti
             if (settings.TemplateFolder != null)
                 builder.Append("-t \"{0}\"", settings.TemplateFolder.FullPath);
 
+            if (settings.GlobalMetadata.Any())
```

Comment:
@stffabi This is part of #31. After closing this and rebasing only https://github.com/cake-contrib/Cake.DocFx/pull/37/commits/f1f40db4471c788c8e29def588ed6c5bd99e8d3b will be part of this PR